torch/csrc/jit/passes/quantization/insert_quant_dequant.cpp

```diff
@@ -1158,6 +1293,7 @@ void InsertQuantDeQuantHelper::runWeightObserver(
   // weight.
   for (const auto& v : weight_values) {
     extractAndRunWeightObserver(module, self, v);
+    insertEmbeddingBagQuantOps(module, self, v);
```

Comment:
why does this happens here?

Reply:
currently we insert prepack/unpack ops here: https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/passes/quantization/finalize.cpp#L50

Reply:
if we need the observer info why can't we insert this op at the same place when we insert quant/dequant ops?

Reply:
That's mainly because the logic where we insert quant/dequant ops doesn't get called for every module instance. I added it here to make sure the prepack and quant ops are added for each embeddingBag module instance.

Reply:
the current logic is we insert for each graph, but the graph itself might be shared by multiple module instances, why that does not work?

Reply:
My understanding was that we cannot have a shared graph here as each module instance will need its own set of pack and quant ops inserted. I initially did take this approach but I noticed that in the final graph only one of the embedding modules had the quant ops inserted.

Reply:
you mean only one of the embedding modules has the packing? or only one has quant + packing and the other one is just ignored?

Reply:
I don't see why we can't have a shared graph here, could you show the problem with putting this together with insert quant/dequant in more detail?

Reply:
@jerryzh168 I think it should be fine. I realized my test graph wasn't doing the right thing. Will update to move to insert quant_dequant function.